contracts/pol/CarbonPOL.sol

```diff
@@ -410,7 +412,9 @@ contract CarbonPOL is ICarbonPOL, Upgradeable, ReentrancyGuardUpgradeable, Utils
 
         // check if the new sale amount is below the current available eth sale amount
         if (newEthSaleAmount < _ethSaleAmount.current) {
```

Comment:
Continuing here: but offering more ETH than there is in the contract shouldn't be allowed, no?

Reply:
This is a kind of tricky question since it's possible that the ETH amount in the contract increases - I think limiting the current amount properly is sufficient for our use case.

Reply:
Also this might lead to unexpected behavior since the goal of the function is to set the eth sale amount - not to set it either to contract balance or eth sale amount.